crates/exex/src/hints.rs

```diff
@@ -0,0 +1,49 @@
+use crate::{db::Database, exex::DATABASE_PATH};
+use cairo_vm::{
+    hint_processor::hint_processor_definition::HintReference,
+    serde::deserialize_program::ApTracking,
+    types::exec_scope::ExecutionScopes,
+    vm::{errors::hint_errors::HintError, vm_core::VirtualMachine},
+    Felt252,
+};
+use rusqlite::Connection;
+use std::collections::HashMap;
+
+/// The hint name for the `print_latest_block_transactions` function.
+///
+/// This is just a string constant that is used to identify the hint in the hint processor.
+pub const KETH_PRINT_TX_HASHES: &str = "print_latest_block_transactions";
+
+/// Prints transaction hashes from the latest block in the SQLite database.
+///
+/// This function connects to an SQLite database specified by `DATABASE_PATH`, retrieves the latest
+/// block, and prints the hash of each transaction contained in that block.
+pub fn print_latest_block_transactions(
```

Comment:
I'd like us to have a `struct` for hints to handle implementation and id together and not have two independents fields

Reply:
Fixed, let me know what you think about this new way of handling the hint